vm/src/stdlib/os.rs

```diff
@@ -1,3 +1,5 @@
+use std::convert::TryFrom;
+use std::convert::TryInto;
```

Comment:
Could you move these `use`s into the body of the function? They're generating a lot of "unused import" warnings since they're only used on `cfg(linux)`

Reply:
Yes, this seems much better.